app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,39 @@
+class CommentsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :find_post, only: [:create]
```

Comment:
here, you can take place the `reply` metod

Reply:
Updated!